package.json

```diff
@@ -4,6 +4,7 @@
   "description": "npm's markdown parser",
   "main": "index.js",
   "scripts": {
+    "prebuild": "node prebuild.js",
```

Comment:
clever, I like the simplicity of this approach; and the fact that you don't need to remember to update variables on each release.